tests/sentinel/tests/00-base.tcl

```diff
@@ -57,16 +57,16 @@ test "SENTINEL PENDING-SCRIPTS returns the information about pending scripts" {
 }
 
 test "SENTINEL PRIMARIES returns a list of monitored primaries" {
-    assert_match "*mymaster*" [S 0 SENTINEL MASTERS]
```

Comment:
i guess we also need some dummy tests that can cover the old commands, otherwise the reply-schemas-validator CI will fail (and we also need it to make sure the old commands can work).

Reply:
Let me do a test on eply-schemas-validator CI and update the result. Thanks BinBin

Reply:
Thanks BinBin，5 sentinel deprecated commands are not hit as below
![image](https://github.com/user-attachments/assets/e565ac0d-908f-4247-a905-79a2cf6338f4)

But I am curious why some many commands are not hit?  it is normal?

Reply:
that is because you just trigger a sentinel-test, it does not run the other tests, so those commands are not hit